doc/getting-started.rst

```diff
@@ -198,6 +209,19 @@ the following file named ``myscript``:
     main :: IO ()
     main = haskellSay "Hello, Haskell!"
 
+.. note::
+
+    Version bounds are recommended but widening or dropping version bound
+    constraints on some packages, especially boot packages like base, for
+    single-file scripts will allow the script to run with a wider range of GHC
+    versions.
```

Comment:
It tries to explain in one sentence a complicated matter with several prerequisites: the reader should know about dependency solving and general and, more importantly, how the process is special for boot libraries (which, more or less, pin versions of GHC). I doubt getting-started is a good place for touching upon such a complex matter at all. But if we have to, the note should be more fair about pros and cons, I think:

```suggestion
    Avoiding version bounds on a boot package, like ``base``, makes running
    the script easier with a different GHC. On the flip side, we lose the information about
    the version of ``base`` (and, therefore, GHC) that the script definitely runs with.
```

Reply:
@ulysses4ever I didn't keep "On the flip side, we lose the information about the version of ``base`` (and, therefore, GHC) that the script definitely runs with". I'm cautious and wary and wouldn't trust any script to definitely run.